recipes/vorpy/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "vorpy" %}
```

Comment:
I think this should be `vorpy3` to match pypi:
https://pypi.org/project/vorpy3/